libcontainer/container_linux.go

```diff
@@ -1043,9 +1043,6 @@ func (c *linuxContainer) isRunning() (bool, error) {
 	}
 	// return Running if the init process is alive
 	if err := syscall.Kill(c.initProcess.pid(), 0); err != nil {
-		if err == syscall.ESRCH {
-			return false, nil
-		}
```

Comment:
this is a known error from kill, means that the process does not exist anymore (in waiting state) for example, so i think the original code makes sense.